benchmark-algorithm/src/main/java/com/project/benchmark/algorithm/dto/order/OrderFiltersTO.java

```diff
@@ -0,0 +1,30 @@
+package com.project.benchmark.algorithm.dto.order;
+
+import com.project.benchmark.algorithm.dto.base.PageParams;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class OrderFiltersTO {
+    private String abbreviation;
+    private boolean active;
+    private Double amount;
+    private Double amountLess;
+    private Double amountMore;
+    private Date creationDateLess; //java.util.Date, możliwe do zmiany
+    private Date creationDateMore; //java.util.Date, możliwe do zmiany
+    private Date dateClosingLess; //java.util.Date, możliwe do zmiany
+    private Date dateClosingMore; //java.util.Date, możliwe do zmiany
+    private Date dateExpirationLess; //java.util.Date, możliwe do zmiany
+    private Date dateExpirationMore; //java.util.Date, możliwe do zmiany
```

Comment:
I think better will be [OffsetDateTime](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/OffsetDateTime.html) (It is from Java 8, while Date is from Java 1.1, so it is quite hard to use)

Reply:
It was resolved yesterday by @lee32147